### PR TITLE
Added rental dto

### PR DESCRIPTION
**Pull request by CodeWebWeaver**

### CODE REVIEW

**Review by IvanFadieiev — DISMISSED** [review]
LGTM!

File: src/main/java/com/jvavateam/carsharingapp/dto/rental/CreateRentalDto.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.jvavateam.carsharingapp.dto.rental;
+
+import java.time.LocalDate;
+
+public record CreateRentalDto(LocalDate rentalDate, Long carId, Long userId) {
+}
```
Comment (Dimagaa) [inline src/main/java/com/jvavateam/carsharingapp/dto/rental/CreateRentalDto.java]: Add validation  for fields

**Review by Dimagaa — CHANGES_REQUESTED** [review]
Good job! Feel free to add field validation for the request dto

File: src/main/java/com/jvavateam/carsharingapp/dto/rental/CreateRentalDto.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.jvavateam.carsharingapp.dto.rental;
+
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+
+public record CreateRentalDto(@NotNull(message = "Rental date cannot be null") LocalDate rentalDate,
```
Comment (Dimagaa) [inline src/main/java/com/jvavateam/carsharingapp/dto/rental/CreateRentalDto.java]: @
```suggestion
public record CreateRentalDto(@NotNull(message = "Rental date cannot be null") @FutureOrPresent LocalDate rentalDate,
```

**Review by Dimagaa — CHANGES_REQUESTED** [review]
Last one

**Review by Dimagaa — APPROVED** [review]

**Review by IvanFadieiev — APPROVED** [review]

**Pull request merged**